[2025-03-25T11:50:29.611+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-25T11:50:29.644+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-01-02T06:00:00+00:00 [queued]>
[2025-03-25T11:50:29.657+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-01-02T06:00:00+00:00 [queued]>
[2025-03-25T11:50:29.658+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 2
[2025-03-25T11:50:29.675+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): ingest> on 2024-01-02 06:00:00+00:00
[2025-03-25T11:50:29.684+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=701) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-25T11:50:29.686+0000] {standard_task_runner.py:72} INFO - Started process 702 to run task
[2025-03-25T11:50:29.688+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_postgres_dag', 'ingest', 'scheduled__2024-01-02T06:00:00+00:00', '--job-id', '189', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres_dag.py', '--cfg-path', '/tmp/tmpy73gheps']
[2025-03-25T11:50:29.691+0000] {standard_task_runner.py:105} INFO - Job 189: Subtask ingest
[2025-03-25T11:50:29.770+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-01-02T06:00:00+00:00 [running]> on host 4c8ab9810a8c
[2025-03-25T11:50:29.885+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_postgres_dag' AIRFLOW_CTX_TASK_ID='ingest' AIRFLOW_CTX_EXECUTION_DATE='2024-01-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-02T06:00:00+00:00'
[2025-03-25T11:50:29.887+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-25T11:50:29.909+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-25T11:50:29.910+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "ls"']
[2025-03-25T11:50:29.929+0000] {subprocess.py:99} INFO - Output:
[2025-03-25T11:50:29.931+0000] {subprocess.py:106} INFO - ls
[2025-03-25T11:50:29.932+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-25T11:50:29.967+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-25T11:50:29.968+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_postgres_dag, task_id=ingest, run_id=scheduled__2024-01-02T06:00:00+00:00, execution_date=20240102T060000, start_date=20250325T115029, end_date=20250325T115029
[2025-03-25T11:50:30.023+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-25T11:50:30.048+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-25T11:50:30.050+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
